test/e2e/kappcontroller/packageinstall_test.go

```diff
@@ -950,3 +950,64 @@ spec:
 		assert.Equal(t, actualSyncPeriod, expectedSyncPeriod)
 	})
 }
+
+
+func TestCELValidation(t *testing.T) {
+	env := e2e.BuildEnv(t)
+	logger := e2e.Logger{}
+	kapp := e2e.Kapp{t, env.Namespace, logger}
+	kubectl := e2e.Kubectl{t, env.Namespace, logger}
+
+	name := "incorrect-spec-without-sa-cluster"
+
+	appYAML := fmt.Sprintf(`
+---
+apiVersion: kappctrl.k14s.io/v1alpha1
+kind: App
```

Comment:
The name of the test file indicates that it's for package installs, should we add a new file for cel validations?

Reply:
I've added this test to a new file, so that in future in any CEL related validation tests to any of the APIs can go in there.